deployment/deployment.go

```diff
@@ -49,8 +49,21 @@ func checkManualReleaseNames(client *houston.Client) bool {
 	return r.Data.GetAppConfig.ManualReleaseNames
 }
 
+// CheckNFSMountDagDeployment returns true when we can set custom NFS location for dags
+func CheckNFSMountDagDeployment(client *houston.Client) bool {
+	req := houston.Request{
+		Query: houston.AppConfigRequest,
+	}
+	r, err := req.DoWithClient(client)
+	if err != nil {
+		return false
+	}
+
+	return r.Data.GetAppConfig.NfsMountDagDeployment
+}
+
 // Create airflow deployment
-func Create(label, ws, releaseName, cloudRole, executor, airflowVersion string, client *houston.Client, out io.Writer) error {
+func Create(label, ws, releaseName, cloudRole, executor, airflowVersion, dagDeploymentType, nfsLocation string, client *houston.Client, out io.Writer) error {
```

Comment:
Is it worth strongly typing `nfsLocation`?  Could this be any string or is there a particular pattern like a `url` that it follows?

Reply:
nope, it can be anytinng like test:/test. So we have lot's of validations in houston side: https://github.com/astronomer/houston-api/blob/fc763126e18c3a9032170a74f738891075a2cddd/src/lib/deployments/validate/config/index.js#L6-L30